### ISSUE TITLE
Making the QoS section a bit less technical

### ISSUE THREAD

**Issue by ghost** [jamulussoftware/jamulussoftware.github.io#748]
Re: [jamulussoftware/jamuluswebsite] Reinhardwh de v4 (PR #747)
@ann0see

Hi, I received an email from GitHub on this topic and can't find where to reply so I opened an Issue.
I modified two files to address this issue: (.diff provided for quick inspection, or cut and paste, or patch)

../wiki/en/Include-QOS-Windows.md.diff
```
--- Include-QOS-Windows.md	2022-05-21 13:46:54.000000000 -0700
+++ Include-QOS-Windows.md	2022-06-10 12:34:39.699702688 -0700
@@ -3,11 +3,11 @@
 
 # Quality of Service (QoS)
 
-Jamulus uses DSCP/CS4 opportunistically to deal with buffer bloat, and uses a default DSCP/CS4 value of 128 (or 0x80). This is compatible with IPv4 and IPv6. Other values can be set using the -Q option, eg -Q [0..255] (where 0 disables QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
+Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, eg -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594.
 
 ## Use of QoS on Windows
 
-Jamulus's QoS settings (including the default) have no effect on Windows because the operating system ignores them. To enable it, you must follow these instructions. Note also that you may need to repeat this procedure every time Jamulus is updated.
+Jamulus' QoS settings (including the default) are set to 0 on recent Windows and Apple because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated.
 
 In Search box beside Start menu Type: Local Group Policy Editor (enter)<br>
 In new window, (click) on the menu icon to display the Actions third panel<br>
```

../wiki/en/Include-Shared-Commands.md.diff
```
--- Include-Shared-Commands.md	2022-05-21 13:46:54.000000000 -0700
+++ Include-Shared-Commands.md	2022-06-10 11:23:02.191837116 -0700
@@ -5,7 +5,7 @@
 - `-i` or `--inifile`   Set location of initialization file (overrides default) 
 - `-n` or `--nogui`    Disable GUI (for use in headless mode)                      
 - `-p` or `--port`    Sets the local UDP port number. Default is 22124 
-- `-Q` or `--qos`     Sets the quality of service DSCP value. Default is 128. Disable with 0. QoS is ignored by Windows. To enable it, see
+- `-Q` or `--qos`     Sets the quality of service DS Field byte. Default is 128 (DSCP/CS4). QoS is ignored by Windows. To enable it, see
 section below
 - `-t` or `--notranslation`   Disable UI language translations
 -  `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
```
[Include-QOS-Windows.new.md](https://github.com/jamulussoftware/jamuluswebsite/files/8882077/Include-QOS-Windows.new.md)
[Include-Shared-Commands.new.md](https://github.com/jamulussoftware/jamuluswebsite/files/8882080/Include-Shared-Commands.new.md)

**Comment by ann0see** [comment]
Thanks!

**Comment by ann0see** [comment]
I think the "however most people will not need..." section should still remain.
Can you open a pull request with these changes?

**Comment by ghost** [comment]
> I think the "however most people will not need..." section should still remain. Can you open a pull request with these changes?

PR #749 is ready for review.

**Comment by ghost** [comment]
PR #750 supersedes the closed PR #749
